src/components/HomepageFeatures/index.tsx

```diff
@@ -32,7 +32,9 @@ const FeatureList: FeatureItem[] = [
         </Link>{" "}
         and can test either the RPM package built in Copr or trigger tests
         directly on different platforms.{" "}
-        <Link to="/docs/configuration/upstream/tests">Learn more.</Link>
+        <Link to="/docs/configuration/upstream/tests">Learn more.</Link> It can
```

Comment:
makes sense to include it in this section, thanks! Maybe we could also link [this](https://packit.dev/posts/openscanhub-prototype) as a sort of docs on setup?